pkgs/development/python-modules/mididings/default.nix

```diff
@@ -0,0 +1,73 @@
+{ lib
+, buildPythonPackage
+, stdenv
+, python
+, fetchFromGitHub
+, pkgs
+, libjack2
+, alsaLib
+, pkgconfig
+, glib
+, boost
+, python36Packages
+}:
+
+buildPythonPackage rec {
```

Comment:
As this is an application please use `buildPythonApplication` and move it out of `python-modules` into a different directory.

Reply:
If it is done as a python application, can you still import it as a library?, cause that is how it is mostly used.